TemplateApplication/Onboarding/HealthKitPermissions.swift

```diff
@@ -11,6 +11,14 @@ import SpeziOnboarding
 import SwiftUI
 
 
+/// Gain access to HealthKit data.
+///
+/// Customize the HealthKitPermissions view to specify your app's usage of HealthKit data to the user.
+///
+/// You can learn more about [Spezi Onboarding in the GitHub repository](https://github.com/StanfordSpezi/SpeziOnboarding)
```

Comment:
Might be a copy and past error here; we should double check this for all files as well 👍
```suggestion
/// You can learn more about [Spezi HealthKit in the GitHub repository](https://github.com/StanfordSpezi/Spezi HealthKit)
```